include/slates_loop.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @brief RL Inference API definition.
+ *
+ * @file slates_loop.h
+ * @author Rajan Chari et al
+ * @date 2018-07-18
+ */
+#pragma once
+#include "action_flags.h"
+#include "base_loop.h"
+#include "err_constants.h"
+#include "factory_resolver.h"
+#include "future_compat.h"
+#include "multi_slot_response.h"
+#include "multi_slot_response_detailed.h"
+#include "rl_string_view.h"
+#include "sender.h"
+
+#include <functional>
+#include <memory>
+
+namespace reinforcement_learning
+{
+class slates_loop : public base_loop
+{
+public:
+  using base_loop::base_loop;
+
```

Comment:
same comment here-> remove RL_DEPRECATED and DEPRECATED mention from comments

Reply:
removed